app/views/centers/index.html.slim

```diff
@@ -0,0 +1,8 @@
+h2 Existing List of PPIS Centers
+table border="1px" 
```

Comment:
when we move this to the /admin, we will have a CSS class available to design a better table